Cargo.toml

```diff
@@ -48,7 +49,9 @@ v2_58 = ["v2_56", "glib-sys/v2_58", "gobject-sys/v2_58"]
 v2_60 = ["v2_58", "glib-sys/v2_60"]
 v2_62 = ["v2_60", "glib-sys/v2_62", "gobject-sys/v2_62"]
 v2_64 = ["v2_62", "glib-sys/v2_64"]
-dox = ["glib-sys/dox", "gobject-sys/dox"]
+bridged_logging = ["log"]
+bridged_logging_domain_macros = ["bridged_logging", "log"]
```

Comment:
The features' name aren't great... :-/

Reply:
Indeed but I don't have a better suggestion. Do you? :)

Reply:
No indeed. Doesn't mean we shouldn't try to find better ones though. :p

Reply:
Other crates seem to just call the feature `log`, so we could do the same. `log` and `log_macros`

Reply:
Sounds good to me!